interfaces/seccomp/template.go

```diff
@@ -549,3 +549,15 @@ pwritev
 # of socket(), bind(), connect(), etc individually.
 socketcall
 `)
+
+// bindSyscallWorkaround specifies the bind() syscall for systems which
+// don't use AppArmor but only seccomp. In those cases the net package
+// from go uses bind for IPv4/IPv6 support detection when loaded. This
+// gets a problem for all hooks when they use snapctl. On systems which
+// use AppArmor the bind() call is properly mediated and the caller gets
+// an error back.
```

Comment:
@jdstrand @niemeyer I've made the comment more precise. Please have another look.